extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/lib/structure/devices/deviceParameters/constants/FunctionalColorGroupEnum.java

```diff
@@ -40,23 +40,23 @@
      * | n/a | Access | Green | Access related functions, door bell |
      *
      */
-    YELLOW(Lists.newArrayList(1)),
-    GREY(Lists.newArrayList(2, 12)),
-    BLUE(Lists.newArrayList(3, 9, 10, 11, 48)),
-    CYAN(Lists.newArrayList(4)),
-    MAGENTA(Lists.newArrayList(5)),
-    BLACK(Lists.newArrayList(8)),
-    WHITE(Lists.newArrayList(-1)),
-    RED(Lists.newArrayList(-2)),
-    GREEN(Lists.newArrayList(-3));
+    YELLOW(Lists.newArrayList((short) 1)),
```

Comment:
Is it really important in terms of sending bits to the devices that you need shorts instead of ints here?

Reply:
That’s the ids we get from digitalSTROM and they are short formatted. We can use int too, but why, when we only need a short?